tests/playbooks/tests_checkpoint_cleanup.yml

```diff
@@ -18,6 +18,10 @@
     - linux-system-roles.network
   tasks:
     - block:
+        # Workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1832897
+        - package:
+            name: dbus-tools
+            state: present
```

Comment:
Is this needed if not using initscripts?  I can't tell if this playbook includes `nm` specific tasks, or perhaps it is needed in either case?
Is this package needed and provided on EL6 through EL8?  Or is this playbook only executed on certain platforms?

Reply:
This playbook does not run on EL6. The test uses both initscripts and nm to trigger an error condition but this only works when dbus-tools is present. initscripts does not really need dbus-tools it is only needed when NM is also running to tell NM not to touch an interface.